quic/s2n-quic-core/events/connection.rs

```diff
@@ -419,3 +410,13 @@ struct DcStateChanged {
     #[nominal_counter("state")]
     state: DcState,
 }
+
+// NOTE - This event MUST come last, since connection-level aggregation depends on it
```

Comment:
You mean last *in this file*? (Asking since you moved it in the file..)

Reply:
Based on s2n-quic-events changes I don't think order matters so approving, but maybe worth fixing up the comment.

Reply:
yeah last in this file. basically i didn't want to have scan everything twice - once to look for all the counters, then to process the connection close.